### ISSUE TITLE
Can't run the playground

### ISSUE THREAD

**Issue by vorant94** [SoCreate/angular-playground#199]
### Versions
```
5.8.4
```

```
Angular CLI: 8.0.1
Node: 10.15.3
OS: darwin x64
Angular: 8.0.0
... animations, cdk, common, compiler, compiler-cli, core, forms
... language-service, material, platform-browser
... platform-browser-dynamic, router

Package                           Version
-----------------------------------------------------------
@angular-devkit/architect         0.800.1
@angular-devkit/build-angular     0.800.1
@angular-devkit/build-optimizer   0.800.1
@angular-devkit/build-webpack     0.800.1
@angular-devkit/core              8.0.1
@angular-devkit/schematics        8.0.1
@angular/cli                      8.0.1
@angular/flex-layout              8.0.0-beta.26
@ngtools/webpack                  8.0.1
@schematics/angular               8.0.1
@schematics/update                0.800.1
rxjs                              6.4.0
typescript                        3.4.5
webpack                           4.30.0
```

### Repro steps

* Step 1 npm install angular-playground
* Step 2 npm run playground

### Observed Behavior
```
Successfully compiled sandbox files.
[ng serve]: ENOENT: no such file or directory, stat '/Users/vorant94/git/kashrut-department-ui/src/tsconfig.app.json'
[ng serve]: 

[ng serve]: Error: ENOENT: no such file or directory, stat '/Users/vorant94/git/kashrut-department-ui/src/tsconfig.app.json'
    at Object.statSync (fs.js:850:3)
    at AngularCompilerPlugin._setupOptions (/Users/vorant94/git/kashrut-department-ui/node_modules/@ngtools/webpack/src/angular_compiler_plugin.js:86:16)
    at new AngularCompilerPlugin (/Users/vorant94/git/kashrut-department-ui/node_modules/@ngtools/webpack/src/angular_compiler_plugin.js:54:14)
    at _createAotPlugin (/Users/vorant94/git/kashrut-department-ui/node_modules/@angular-devkit/build-angular/src/angular-cli-files/models/webpack-configs/typescript.js:72:12)
    at Object.getNonAotConfig (/Users/vorant94/git/kashrut-department-ui/node_modules/@angular-devkit/build-angular/src/angular-cli-files/models/webpack-configs/typescript.js:78:19)
    at getCompilerConfig (/Users/vorant94/git/kashrut-department-ui/node_modules/@angular-devkit/build-angular/src/browser/index.js:77:95)
    at webpack_browser_config_1.generateBrowserWebpackConfigFromContext.wco (/Users/vorant94/git/kashrut-department-ui/node_modules/@angular-devkit/build-angular/src/browser/index.js:53:9)
    at scriptTargets.map.scriptTarget (/Users/vorant94/git/kashrut-department-ui/node_modules/@angular-devkit/build-angular/src/utils/webpack-browser-config.js:56:26)
    at Array.map (<anonymous>)
    at generateWebpackConfig (/Users/vorant94/git/kashrut-department-ui/node_modules/@angular-devkit/build-angular/src/utils/webpack-browser-config.js:31:26)
    at process._tickCallback (internal/process/next_tick.js:68:7)
    at Function.Module.runMain (internal/modules/cjs/loader.js:757:11)
    at startup (internal/bootstrap/node.js:283:19)
    at bootstrapNodeJSCore (internal/bootstrap/node.js:622:3)
[ng serve]: 
```

### Desired Behavior
```
compile files and run dev server
```

**Comment by lurock** [comment]
Have you tried installing it with these instructions:  https://angularplayground.it/docs/getting-started/angular-cli

**Comment by walrus95** [comment]
> Have you tried installing it with these instructions: https://angularplayground.it/docs/getting-started/angular-cli Using "ng add angular-playground" may solve your issue.

I'm experiencing the exact same issue, and I did use "ng add angular-playground" in setting up my project.

**Comment by poul-kg** [comment]
Is there way to configure paths for `tsconfig.json`, `tsconfig.app.json` etc, files? Looks like this is the issue.

**Comment by n-gottschalt** [comment]
Angular 8 moved the location of tsconfig.*.json to the root. I got it to work by updating the playground project in angular.json to point to this new location.

**Comment by arroyocode** [comment]
Thanks @n-gottschalt  @poul-kg for the tip. Acknowledge the issue and are working on a fix.

**Comment by arroyocode** [comment]
Issue fixed in 6.0.1 merged today.
